server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectWriteAheadLogs.java

```diff
@@ -80,10 +80,10 @@
 
 public class GarbageCollectWriteAheadLogs {
   private static final Logger log = LoggerFactory.getLogger(GarbageCollectWriteAheadLogs.class);
+  private static final Map<HostAndPort,Long> firstSeenDead = new HashMap<HostAndPort,Long>();
 
   private final AccumuloServerContext context;
   private final VolumeManager fs;
-  private final Map<HostAndPort,Long> firstSeenDead = new HashMap<HostAndPort,Long>();
```

Comment:
I'm curious if there are tests to ensure that state is eventually cleared if the tserver comes back online. I'd be concerned about rollover if the tserver came back.